src/otx/core/data/dataset/base.py

```diff
@@ -141,7 +142,11 @@ def __getitem__(self, index: int) -> T_OTXDataEntity:
         msg = f"Reach the maximum refetch number ({self.max_refetch})"
         raise RuntimeError(msg)
 
-    def _get_img_data_and_shape(self, img: Image) -> tuple[np.ndarray, tuple[int, int]]:
+    def _get_img_data_and_shape(
```

Comment:
Could you add a unit test for the roi case?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/4105/commits/97255e3b5ccb5581b15bac882eba749d94da2c6f

Reply:
Thanks!